### DISCUSSION TITLE
자리비움 후 복귀 시 튕김 현상

### DISCUSSION THREAD

**Discussion by joorijung** [metamong-friends/metamong-friends.github.io#50] [제보하기]
자리비움 후 다시 켜면 튕겨서 재접속 해야하는 것 같아요!

**Comment by ACT-TJ** [comment]
안녕하세요! 

저희 함께 회의할 때 저도 제 옆에 있던 캐릭터가 튕겨져 나가는 걸 보았습니다 ㅠㅠ
다만 자리비움 후 다시 킨다고 해서 재발하고 있지는 않는 상황입니다.
자리비움 - 복귀 관련해서 테스트를 더 진행하여 재현되는 케이스를 찾아보고
로직 점검 추가로 해보도록 하겠습니다.

혹시 재현케이스를 좀 더 특정할 수 있게 된다면 추가 코멘트 주시면 큰 도움이 될 것 같습니다.

감사합니다!!
